### ISSUE TITLE
Can't build plugin anymore

### ISSUE THREAD

**Issue by Simbud-7** [senx/warp10-plugin-kafka#4]
Hi,

I used to build successfully the plugin in my CI, but since few days, it doesn't build anymore as Maven has removed their .pom :

```
Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'warp10-plugin-kafka'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find org.ajoberstar.grgit:grgit-core:4.0.1.
     Searched in the following locations:
       - https://plugins.gradle.org/m2/org/ajoberstar/grgit/grgit-core/4.0.1/grgit-core-4.0.1.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project : > net.nemerosa.versioning:net.nemerosa.versioning.gradle.plugin:2.14.0 > gradle.plugin.net.nemerosa:versioning:2.14.0
```

I didn't find where to change the version to 4.1.1 as 4.0.1 has just been removed : https://repo1.maven.org/maven2/org/ajoberstar/grgit/grgit-core/

FYI, I'm using this method : https://github.com/senx/warp10-plugin-kafka#from-source

Thanks for any help / workaround :)

**Comment by Giwi** [comment]
Hi, this lib depends on net.nemerosa.versioning:2.14.0 and is hosted on Bintray (RIP), this Gradle plugin has been updated, but  do not fit (yet) our CI/CD requirements. We will use another way to fetch the git tag/build number.

There is 2 workarounds:
- grab our compiled plugin directly here: https://warpfleet.senx.io/browse/io.warp10/warp10-plugin-kafka 
- use my fork (for testing purpose only!!) https://github.com/Giwi/warp10-plugin-kafka before the merge of my PR.

**Comment by Simbud-7** [comment]
Thanks for your answer & work-around.

As it's for using in a CI/CD, I'll take the compiled plugin directly from the website and store it in our artifacts registry, as I guess the file link will be removed when a newer version is released.

**Comment by Giwi** [comment]
Nope, the file link is a Maven link, so, you can still download the version you want: https://repo.maven.apache.org/maven2/io/warp10/warp10-plugin-kafka/

You can also use our Gradle plugin in order to fetch the right version (and its dependencies): https://warpfleet.senx.io/doc

**Comment by Simbud-7** [comment]
Got it.
Thanks for your help and answer

**Comment by Giwi** [comment]
You are welcome :)

**Comment by Simbud-7** [comment]
Hi,

I tried both available plugins : 1.0.1 and 2.0.0 retrieved here : https://repo.maven.apache.org/maven2/io/warp10/warp10-plugin-kafka/

And I got the same error twice (Warp10 2.11.1) :

```
Exception in thread "main" java.lang.RuntimeException: java.lang.RuntimeException: Kafka Plugin not compatible with your Warp 10 version.
```

**Comment by Giwi** [comment]
Strange, 
1.0.1-uberjar is for Warp 10 2.x
2.0.0-uberjar is for Warp 10 3.x -> this version prints the message "Kafka Plugin not compatible with your Warp 10 version."
be sure you do not have both libs in the same folder.

**Comment by Simbud-7** [comment]
good catch, both were in the same folder. Thanks !
